docs/GettingStarted.md

```diff
@@ -326,26 +256,28 @@ Net::HTTP module.
 
 Experimental distributed tracing support is available for this library.
 By default, this is disabled. You need to enable it, either on a per-connection basis,
-by setting the ``:distributed_tracing_enabled`` config entry to ``true`` using
+by setting the ``:distributed_tracing`` config entry to ``true`` using
```

Comment:
"Experimental distributed tracing" should probably get its own subheading, since its a major feature underneath HTTP.